src/solvers/dg_unstructured_quad/sort_boundary_conditions.jl

```diff
@@ -9,42 +9,55 @@ set by the user in the elixir file is also stored for printing.
 !!! warning "Experimental code"
     This boundary condition container is experimental and can change any time.
 """
-struct UnstructuredQuadSortedBoundaryTypes{N, BCs<:NTuple{N, Any}}
+mutable struct UnstructuredQuadSortedBoundaryTypes{N, BCs<:NTuple{N, Any}}
   boundary_condition_types::BCs # specific boundary condition type(s), e.g. BoundaryConditionWall
   boundary_indices::NTuple{N, Vector{Int}} # integer vectors containing global boundary indices
   boundary_dictionary::Dict{Symbol, Any} # boundary conditions as set by the user in the elixir file
 end
 
+@inline nboundaries(container::UnstructuredQuadSortedBoundaryTypes) = sum(container.boundary_indices .|> length)
```

Comment:
```suggestion
@inline nboundaries(container::UnstructuredQuadSortedBoundaryTypes) = sum(length, container.boundary_indices)
```
Should do the same without allocating anything.